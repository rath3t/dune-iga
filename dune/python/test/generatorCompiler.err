###############################
###cmake --build /dune/dune-common/build-cmake/dune-env/.cache/dune-py --target NURBSGrid_cc2260334160fd59946b2e64fd441123 --parallel --
In file included from /tmp/dune-iga/dune/iga/nurbsgrid.hh:20,
                 from /tmp/dune-iga/dune/python/iga/reader.hh:12,
                 from /dune/dune-common/build-cmake/dune-env/.cache/dune-py/python/dune/generated/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc:8:
/tmp/dune-iga/dune/iga/gridcapabilities.hh:78:8: warning: partial specialization ‘struct Dune::Capabilities::canCommunicate<Dune::IGA::NURBSGrid<dim, dimworld, ScalarType>, codim>’ is not more specialized than
   78 | struct canCommunicate< Dune::IGA::NURBSGrid<dim, dimworld, ScalarType> , codim >
      |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /dune/dune-grid/dune/grid/common/grid.hh:27,
                 from /dune/dune-grid/dune/grid/common/intersection.hh:9,
                 from /dune/dune-grid/dune/grid/io/file/dgfparser/gridptr.hh:25,
                 from /dune/dune-grid/dune/grid/io/file/dgfparser/dgfparser.hh:30,
                 from /tmp/dune-iga/dune/iga/io/ibra/ibrareader.hh:15,
                 from /tmp/dune-iga/dune/python/iga/reader.hh:6:
/dune/dune-grid/dune/grid/common/capabilities.hh:96:12: note: primary template ‘template<class Grid, int codim> struct Dune::Capabilities::canCommunicate’
   96 |     struct canCommunicate
      |            ^~~~~~~~~~~~~~

###############################
