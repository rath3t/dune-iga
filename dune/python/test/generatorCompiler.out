###############################
###cmake --build /dune/dune-common/build-cmake/dune-env/.cache/dune-py --target NURBSGrid_cc2260334160fd59946b2e64fd441123 --parallel --
[ 50%] Building CXX object python/dune/generated/CMakeFiles/NURBSGrid_cc2260334160fd59946b2e64fd441123.dir/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc.o
/usr/bin/c++ -std=c++20  -DENABLE_CLIPPERLIB2=1 -DENABLE_EARCUT=1 -DENABLE_EIGEN=1 -DENABLE_MPI=1 -DENABLE_NLOHMANNJSON=1 -DENABLE_QUADMATH=1 -DENABLE_SUITESPARSE=1 -DENABLE_SUPERLU=1 -DENABLE_VTK_ZLIB=1 -DHAVE_CONFIG_H -DMETIS_API_VERSION=5 -DModelP -DNURBSGrid_cc2260334160fd59946b2e64fd441123_EXPORTS -D_GLIBCXX_USE_FLOAT128 -I/dune/dune-common/build-cmake/dune-env/.cache/dune-py -I/usr/include/python3.10 -I/dune/dune-common -I/dune/dune-uggrid -I/dune/dune-geometry -I/dune/dune-typetree -I/dune/dune-istl -I/dune/dune-localfunctions -I/dune/dune-grid -I/dune/dune-foamgrid -I/dune/dune-alugrid -I/dune/dune-functions -I/dune/dune-vtk -I/tmp/dune-iga -isystem /usr/lib/x86_64-linux-gnu/openmpi/include -isystem /usr/lib/x86_64-linux-gnu/openmpi/include/openmpi -isystem /usr/local/include/eigen3 -isystem /usr/include/suitesparse -isystem /usr/include/superlu -std=c++20 -O3 -DNDEBUG -fPIC -fext-numeric-literals -MD -MT python/dune/generated/CMakeFiles/NURBSGrid_cc2260334160fd59946b2e64fd441123.dir/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc.o -MF CMakeFiles/NURBSGrid_cc2260334160fd59946b2e64fd441123.dir/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc.o.d -o CMakeFiles/NURBSGrid_cc2260334160fd59946b2e64fd441123.dir/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc.o -c /dune/dune-common/build-cmake/dune-env/.cache/dune-py/python/dune/generated/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc
[100%] Linking CXX shared library NURBSGrid_cc2260334160fd59946b2e64fd441123.so
/usr/bin/c++ -std=c++20  -fPIC -std=c++20 -O3 -DNDEBUG -shared -Wl,-soname,NURBSGrid_cc2260334160fd59946b2e64fd441123.so -o NURBSGrid_cc2260334160fd59946b2e64fd441123.so CMakeFiles/NURBSGrid_cc2260334160fd59946b2e64fd441123.dir/NURBSGrid_cc2260334160fd59946b2e64fd441123.cc.o -Wl,-rpath,/dune/dune-vtk/build-cmake/lib:/dune/dune-alugrid/build-cmake/lib:/dune/dune-grid/build-cmake/lib:/dune/dune-geometry/build-cmake/lib:/dune/dune-uggrid/build-cmake/lib:/dune/dune-common/build-cmake/lib:/usr/lib/x86_64-linux-gnu/openmpi/lib:/usr/local/lib /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libsuperlu.so /usr/lib/x86_64-linux-gnu/libmetis.so /usr/lib/x86_64-linux-gnu/libopenblas.so -lm -ldl /dune/dune-vtk/build-cmake/lib/libdunevtk.so /dune/dune-alugrid/build-cmake/lib/libdunealugrid.so /dune/dune-grid/build-cmake/lib/libdunegrid.so /dune/dune-geometry/build-cmake/lib/libdunegeometry.so /dune/dune-uggrid/build-cmake/lib/libduneuggrid.so /dune/dune-common/build-cmake/lib/libdunecommon.so /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libz.so /usr/lib/x86_64-linux-gnu/libsuperlu.so /usr/lib/x86_64-linux-gnu/libmetis.so /usr/lib/x86_64-linux-gnu/libopenblas.so -lm -ldl -lm /usr/lib/x86_64-linux-gnu/openmpi/lib/libmpi.so /usr/local/lib/libClipper2.so /usr/lib/x86_64-linux-gnu/libldl.so /usr/lib/x86_64-linux-gnu/libspqr.so -lm -ldl /usr/lib/x86_64-linux-gnu/libumfpack.so /usr/lib/x86_64-linux-gnu/libcholmod.so /usr/lib/x86_64-linux-gnu/libamd.so /usr/lib/x86_64-linux-gnu/libcolamd.so /usr/lib/x86_64-linux-gnu/libcamd.so /usr/lib/x86_64-linux-gnu/libccolamd.so /usr/lib/x86_64-linux-gnu/libsuitesparseconfig.so /usr/lib/x86_64-linux-gnu/libopenblas.so -lquadmath
[100%] Built target NURBSGrid_cc2260334160fd59946b2e64fd441123

###############################
[100%] Built target referenceelements_de274310a0055100b832a0e8892b0258
[100%] Built target referenceelements_0fff57b283123cc055345ec95574c857
[100%] Built target referenceelements_f4457f35395151d83a965d4cf5fce70d
